mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java

```diff
@@ -72,7 +72,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         final String qrData = getIntent().getStringExtra(Constants.QR_DATA);
         mShowQrPresenter.generateQr(qrData);
-        tvQrData.setText(qrData);
+        tvQrData.setText(getString(R.string.email) + ": " + qrData);
```

Comment:
Only keep this line, other two should not be changed as they are used for generating QR code and it would disturb the processing of scanning of QR code.

Reply:
Okay sir